src/pages/resultsView/SurvivalStoreHelper.ts

```diff
@@ -50,3 +51,46 @@ export function getPatientSurvivals(
         return [];
     }
 }
+
+export function getPatientSurvivalUniqueData(
```

Comment:
this function returns an array of strings but it's not super clear what they are based on function name _Data.  status values?    I also wonder if it would be better to return array of the clinical data objects and then having calling code pluck off the values.  This makes this more reausable.